res/css/views/rooms/_AppsDrawer.pcss

```diff
@@ -204,13 +204,15 @@ limitations under the License.
         white-space: nowrap;
         overflow: hidden;
         text-overflow: ellipsis;
+        margin-left: 9px;
+        pointer-events: none;
 
-        .mx_WidgetAvatar {
+        .mx_AppTileMenuBar_title_widgetAvatar {
             margin-right: 12px;
         }
 
-        > :last-child {
```

Comment:
`.mx_AppTileMenuBar_title > :last-child` was [this `span` element](https://github.com/matrix-org/matrix-react-sdk/pull/10895/files#diff-0a893165e4d1987c581fd62cbf23363bc9412783e5cb24ac2fba68d2a2a5bba7L511) below.